cli/docker.go

```diff
@@ -0,0 +1,304 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	cfg "github.com/flynn/flynn/cli/config"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+	"github.com/flynn/flynn/host/types"
+)
+
+func init() {
+	register("docker", runDocker, `
+usage: flynn docker set-push-url <url>
```

Comment:
This should have a default URL (`flynn docker set-push-url` should work with no arguments)